api/projects/projects.py

```diff
@@ -87,6 +95,13 @@ async def get_project(
     user.check_project_access(project_name)
     coalesce = RequestCoalescer()
     project = await coalesce(ProjectEntity.load, project_name)
+
+    if "productTypes" not in project.config:
+        project.config["productTypes"] = {}
+
+    if "default" not in project.config["productTypes"]:
```

Comment:
I though the default would not be stored next to other product types. I know it probably won't happen, but this might collide with `"default"` product type. Maybe issue for future us when it happens?